script/scaffold/templates/config_flow/tests/test_config_flow.py

```diff
@@ -1,12 +1,12 @@
 """Test the NEW_NAME config flow."""
 from unittest.mock import patch
 
+from asynctest import patch as asyncpatch
```

Comment:
I think should import this as `patch` and remove the unittest.mock patch import. That's one of the nice things about it, that it handles both normal functions and coroutines.

Eventually when we migrate back to unittest.mock patch, the diff will be cleaner too if we call it `patch`.